tokio/src/sync/mpsc/bounded.rs

```diff
@@ -475,6 +475,87 @@ impl<T> Receiver<T> {
     pub fn poll_recv(&mut self, cx: &mut Context<'_>) -> Poll<Option<T>> {
         self.chan.recv(cx)
     }
+
+    /// Polls to receive multiple messages on this channel, extending the provided buffer.
+    ///
+    /// This method attempts to receive messages from the channel and store them in the provided buffer,
+    /// up to the specified `limit`. It behaves similarly to `poll_recv` but for multiple messages. If `limit`
+    /// is zero, the function returns immediately with `0`.
+    ///
+    /// This method also shares similarities with `recv_many`, including its behavior in response to channel closure
+    /// and message availability. For `limit > 0`, if there are no messages in the channel's queue,
+    /// but the channel has not yet been closed, this method will sleep until a message is sent or
+    /// the channel is closed. The channel is  closed when all senders have been dropped, or when `close` is called.
```

Comment:
These two paragraphs can probably just be deleted. As long as the list below is comprehensive, this isn't necessary, and it is obvious that it is similar to `poll_recv` and `recv_many`.

Reply:
Sure & deleted